databricks_cli/sdk/api_client.py

```diff
@@ -113,10 +113,10 @@ def perform_query(self, method, path, data = {}, headers = None):
             if method == 'GET':
                 translated_data = {k: _translate_boolean_to_query_param(data[k]) for k in data}
                 resp = self.session.request(method, self.url + path, params = translated_data,
```

Comment:
Is it possible to revert all changes to this file?

Reply:
Reverted back to old file